service/waterfall.go

```diff
@@ -619,9 +619,16 @@ func (uis *UIServer) waterfallPage(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	newUILink := ""
-	if len(uis.Settings.Ui.UIv2Url) > 0 {
-		newUILink = fmt.Sprintf("%s/commits/%s", uis.Settings.Ui.UIv2Url, project.Identifier)
+	newUILink := fmt.Sprintf("%s/commits/%s", uis.Settings.Ui.UIv2Url, project.Identifier)
+
+	u := gimlet.GetUser(r.Context())
+	if u != nil {
+		usr, ok := u.(*user.DBUser)
+		if ok {
+			if usr.BetaFeatures.SpruceWaterfallEnabled {
+				newUILink = fmt.Sprintf("%s/project/%s/waterfall", uis.Settings.Ui.UIv2Url, project.Identifier)
+			}
+		}
```

Comment:
Can we make the default the waterfall page and only redirect to the commits page if they're _not_ opted in to the beta? Since the beta period will be ending ~this week I think that will facilitate moving everyone to the waterfall (and simplify this cleanup by making the `if` statement deletable)

Reply:
I can reverse the if-statements! But I'm a bit confused about the admin feature condition; on the frontend we implemented it so that if the admin feature is disabled, it turns off the waterfall page. It seems like we might be doing the opposite here which could be confusing

Another option if you want is to just increase friction and always link people to the new Waterfall page, which might annoy them, but that's okay

Reply:
(to clarify I'm just not sure what to do with admin settings, but I can definitely access them here in 	`uis.Settings.Ui.BetaFeatures.SpruceWaterfallEnabled`!)